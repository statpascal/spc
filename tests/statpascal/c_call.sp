program c_call;

function pow (x, y: real): real; external;
procedure printf (var p: char; a, b: integer); external;

var 
    fmt: string;

begin
    writeln (pow (2, 3):3:0, ' ', pow (2, 0.5):8:6);
    fmt := 'The values are %d and %d' + chr (10) + chr (10);
    printf (fmt [1], 2, 3)
end.
