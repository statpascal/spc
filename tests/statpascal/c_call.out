  8 1.414214
The values are 2 and 3

